src/js/components/TaskDetailComponent.jsx

```diff
@@ -38,6 +38,23 @@ var TaskDetailComponent = React.createClass({
     );
   },
 
+  getTaskEndpoints: function () {
+    var props = this.props;
+    var task = props.task;
+    if (task.ports == null || task.ports.length === 0) {
+      return (<dd>None</dd>);
+    }
+
+    return task.ports.map((port) => {
+      let endpoint = `${task.host}:${port}`;
+      return (
+        <dd key={endpoint} className="overflow-ellipsis">
+          <a href={endpoint} target="_blank">{endpoint}</a>
```

Comment:
Exactly, that is my though, too. :) If we are consequent, this shouldn't be links.